### ISSUE TITLE
yarn install --production gives error

### ISSUE THREAD

**Issue by milkcocoa** [ColorlibHQ/AdminLTE#2299]
### Issue type:
- [ ] Feature request 
- [x] Bug report       
- [ ] Documentation   

### Environment:
* AdminLTE Version:  3.0.0-rc3
* Operating System:  Google Cloud Build
* Browser (Version):  

### Description:

I use it under Google Cloud Build environment, and RC 3 prevents yarn from building our package.json. Google Cloud Build executes `yarn install --production` and gives the following error.

```
Step #1 - "builder": error /workspace/node_modules/admin-lte: Command failed.
Step #1 - "builder": Exit code: 1
Step #1 - "builder": Command: npm run plugins
Step #1 - "builder": Arguments: 
Step #1 - "builder": Directory: /workspace/node_modules/admin-lte
Step #1 - "builder": Output:
Step #1 - "builder": > admin-lte@3.0.0-rc.3 plugins /workspace/node_modules/admin-lte
Step #1 - "builder": > node build/npm/Publish.js -v
Step #1 - "builder": 
Step #1 - "builder": internal/modules/cjs/loader.js:638
Step #1 - "builder":     throw err;
Step #1 - "builder":     ^
Step #1 - "builder": 
Step #1 - "builder": Error: Cannot find module 'fs-extra'
Step #1 - "builder":     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
Step #1 - "builder":     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
Step #1 - "builder":     at Module.require (internal/modules/cjs/loader.js:692:17)
Step #1 - "builder":     at require (internal/modules/cjs/helpers.js:25:18)
Step #1 - "builder":     at Object.<anonymous> (/workspace/node_modules/admin-lte/build/npm/Publish.js:2:17)
Step #1 - "builder":     at Module._compile (internal/modules/cjs/loader.js:778:30)
Step #1 - "builder":     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
Step #1 - "builder":     at Module.load (internal/modules/cjs/loader.js:653:32)
Step #1 - "builder":     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
Step #1 - "builder":     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
Step #1 - "builder": npm ERR! code ELIFECYCLE
Step #1 - "builder": npm ERR! errno 1
Step #1 - "builder": npm ERR! admin-lte@3.0.0-rc.3 plugins: `node build/npm/Publish.js -v`
Step #1 - "builder": npm ERR! Exit status 1
Step #1 - "builder": npm ERR! 
Step #1 - "builder": npm ERR! Failed at the admin-lte@3.0.0-rc.3 plugins script.
Step #1 - "builder": npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Step #1 - "builder": npm WARN Local package.json exists, but node_modules missing, did you mean to install?
Step #1 - "builder": 
Step #1 - "builder": npm ERR! A complete log of this run can be found in:
Step #1 - "builder": npm ERR!     /builder/home/.npm/_logs/2019-10-03T06_05_23_169Z-debug.log
Step #1 - "builder": 
Step #1 - "builder": ERROR    error: `yarn_install` returned code: 1
Step #1 - "builder": INFO     yarn_install took 66 seconds
Step #1 - "builder": INFO     building_packages_json_layer took 66 seconds
Step #1 - "builder": INFO     build process for FTL image took 67 seconds
Step #1 - "builder": INFO     full build took 67 seconds
Step #1 - "builder": ERROR    `yarn_install` had stderr output:
Step #1 - "builder": warning "admin-lte > tempusdominus-bootstrap-4@5.1.2" has unmet peer dependency "moment-timezone@^0.5.11".
Step #1 - "builder": warning "admin-lte > tempusdominus-bootstrap-4@5.1.2" has unmet peer dependency "tempusdominus-core@5.0.3".
Step #1 - "builder": warning " > sass-loader@7.3.1" has unmet peer dependency "webpack@^3.0.0 || ^4.0.0".
Step #1 - "builder": error /workspace/node_modules/admin-lte: Command failed.
Step #1 - "builder": Exit code: 1
Step #1 - "builder": Command: npm run plugins
Step #1 - "builder": Arguments: 
Step #1 - "builder": Directory: /workspace/node_modules/admin-lte
Step #1 - "builder": Output:
Step #1 - "builder": > admin-lte@3.0.0-rc.3 plugins /workspace/node_modules/admin-lte
Step #1 - "builder": > node build/npm/Publish.js -v
Step #1 - "builder": 
Step #1 - "builder": internal/modules/cjs/loader.js:638
Step #1 - "builder":     throw err;
Step #1 - "builder":     ^
Step #1 - "builder": 
Step #1 - "builder": Error: Cannot find module 'fs-extra'
Step #1 - "builder":     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
Step #1 - "builder":     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
Step #1 - "builder":     at Module.require (internal/modules/cjs/loader.js:692:17)
Step #1 - "builder":     at require (internal/modules/cjs/helpers.js:25:18)
Step #1 - "builder":     at Object.<anonymous> (/workspace/node_modules/admin-lte/build/npm/Publish.js:2:17)
Step #1 - "builder":     at Module._compile (internal/modules/cjs/loader.js:778:30)
Step #1 - "builder":     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
Step #1 - "builder":     at Module.load (internal/modules/cjs/loader.js:653:32)
Step #1 - "builder":     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
Step #1 - "builder":     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
Step #1 - "builder": npm ERR! code ELIFECYCLE
Step #1 - "builder": npm ERR! errno 1
Step #1 - "builder": npm ERR! admin-lte@3.0.0-rc.3 plugins: `node build/npm/Publish.js -v`
Step #1 - "builder": npm ERR! Exit status 1
Step #1 - "builder": npm ERR! 
Step #1 - "builder": npm ERR! Failed at the admin-lte@3.0.0-rc.3 plugins script.
Step #1 - "builder": npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
Step #1 - "builder": npm WARN Local package.json exists, but node_modules missing, did you mean to install?
```

I've tried the same command under my local environment, and got the same one.

**Comment by REJack** [comment]
I've added a fix in my fork and push it soon to the repo.

**Comment by REJack** [comment]
This fix is released with v3.0.0-rc.4.
